net/relay.go

```diff
@@ -68,6 +93,9 @@ func (h *Host) handleRelayStream(stream libp2pnetwork.Stream) {
 
 // SubmitClaimToRelayer sends a request to relay a swap claim to a peer.
 func (h *Host) SubmitClaimToRelayer(relayerID peer.ID, request *RelayClaimRequest) (*RelayClaimResponse, error) {
+	// The timeout should be short enough, that the Maker can try multiple relayers
+	// before T1 expires even if the receiving node accepts the relay request and
+	// just sits on it without doing anything.
```

Comment:
note that the timeout here is specifically for connecting and opening the stream, not for receiving the response. we'd need a different timer for that

Reply:
Comment updated with a link to the issue to fix the problem across all 3 streams.